### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by 0x00b1**

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -8,6 +8,19 @@ possible.
 We will periodically release updates to the core `neuralcompression` package to
 PyPI.
 
+## Building
+
```
Comment (mmuckley) [inline .github/CONTRIBUTING.md:12]:
```suggestion

Using `pip`, you can install the package in development mode by running:

```

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -8,6 +8,19 @@ possible.
...
+```sh
+pip install -e .
+pip install -r dev-requirements.txt
+```
+
+## Testing
+
```
Comment (mmuckley) [inline .github/CONTRIBUTING.md:19]:
```suggestion

We test the package using `pytest`, which you can run locally by typing

```

**Review by mmuckley — CHANGES_REQUESTED** [review]
A few minor comments to clarify what the code is doing.

**Comment by 0x00b1** [comment]
Lovely! Accepted both!

**Review by mmuckley — APPROVED** [review]
LGTM.

**Pull request merged**